### DISCUSSION TITLE
Stepping in Autohotkey Plus Plus no longer works

### DISCUSSION THREAD

**Discussion by siljuberg** [mark-wiemer/ahkpp#154] [General]
Hi. I've been using this great plugin for about a year now. It has worked fine up until today, when I no longer can step through my code with it. I am able to put a break point and it gets hit, but then when I try to go one step it exits/stops. I can also put a break point later in my code and it will stop at that breakpoint.

I don't remember exactly when it worked last. But for sure it worked 17th of June.

Could someone please help? Anything to try out?

**VSCode information**
Version: 1.68.1 (system setup)
Commit: 30d9c6cd9483b2cc586687151bcbcd635f373630
Date: 2022-06-14T12:48:58.283Z
Electron: 17.4.7
Chromium: 98.0.4758.141
Node.js: 16.13.0
V8: 9.8.177.13-electron.0
OS: Windows_NT x64 10.0.19043

**Comment by fade2gray** [comment]
It sounds like you applied the v2.8.3 update. Revert back to v2.8,2 by clicking the extensions setting's cogwheel and selecting `Install Another Version` from the menu.

#153

**Reply by siljuberg** [reply]
Thanks! That did the trick:)

**Reply by mark-wiemer** [reply]
I've republished 2.8.2 as 2.8.4, again this should not have happened and I'm sorry. I've learned my lesson and I'll definitely do my due diligence before future code changes. Sorry for the inconvenience, and thank you both for your patience and quick reports, @fade2gray  and @siljuberg :) you saved a lot of others lots of trouble
